### ISSUE TITLE
Build failures when running electron-rebuild

### ISSUE THREAD

**Issue by jlao** [grpc/grpc-node#301]
I'm getting the following build errors when I attempt to run `electron-rebuild` with grpc.

```
../deps/grpc/third_party/boringssl/crypto/curve25519/spake25519.c:26:10: fatal error: '../../third_party/fiat/internal.h' file not found
#include "../../third_party/fiat/internal.h"
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
```

Here are the versions I'm using:

```
  "dependencies": {
    "grpc": "^1.11.0"
  },
  "devDependencies": {
    "electron": "^1.8.6",
    "electron-rebuild": "^1.7.3"
  }
```

**Comment by nicolasnoble** [comment]
Generally speaking, please don't use electron-rebuild with grpc. It generally messes things up due to it ignoring our installation steps, and not calling into node-pre-gyp.

The best way at that point to get grpc running into Electron is to run the following command:

```
npm rebuild grpc --runtime=electron --target=1.8.6 --dist-url=https://atom.io/download/electron
```

That being said, you're getting an error I am not expecting. @murgatroid99 are we sure we packaged sources properly in the npm package ?

**Comment by murgatroid99** [comment]
I see the problem. Our `package.json` doesn't have a generated list of files because that was making npm unhappy. It has a manually curated list of globs, and it doesn't include anything from `grpc/third_party/boringssl/third_party`.

**Comment by nicolasnoble** [comment]
Fixed by #304.
